src/java/entities/PerishableFood.java

```diff
@@ -3,31 +3,45 @@
 import java.time.LocalDate;
 
 public class PerishableFood extends Food {
-    private boolean isExpired;
+    private final boolean isExpired;
```

Comment:
why would you make this final??  the point is that we want to change this based on whether or not it's expired